lib/internal/cluster/master.js

```diff
@@ -70,6 +79,15 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
+  const hasDebugArg = process.execArgv.some((argv) => {
```

Comment:
Neither `process._inspectorEnbaled` or `inspector.url()` work for this purpose, because they don't catch `--inspect-port`. That option doesn't "enable" the inspector (turn it on), it just changes the default port in case it is turned on later. So, the regex is necessary, AFAICT.